### PR TITLE
fix: base-ref/head-ref missed in dependency-check-ci on branch push

### PR DESCRIPTION
**Pull request by liuxiaocs7**
refer to https://github.com/apache/incubator-hugegraph/pull/2308

### CODE REVIEW

**Review by simon824 — APPROVED** [review]
LGTM

File: .github/workflows/check-dependencies.yml [hunk 1]
```diff
@@ -1,10 +1,6 @@
 name: "3rd-party check"
 
 on:
-  push:
-    branches:
-      - main
-      - 'release-*'
```
Comment (simon824) [inline .github/workflows/check-dependencies.yml:7]: keep consistent with the `hugegraph`?
https://github.com/apache/incubator-hugegraph/blob/d4b95ca32371dfdee60d448fa3e5ab38fc4e288e/.github/workflows/check-dependencies.yml#L6

Comment (liuxiaocs7) [reply .github/workflows/check-dependencies.yml:7]: Thanks, i'll take a look, just ref: https://github.com/apache/incubator-hugegraph-toolchain/pull/516 before.

Comment (imbajin) [reply .github/workflows/check-dependencies.yml:7]: > Thanks, i'll take a look, just ref: [apache/incubator-hugegraph-toolchain#516](https://github.com/apache/incubator-hugegraph-toolchain/pull/516) before.

The reason is the running ci **context is a little different**, we need `Check License Header` in `push` action but don't need with `Dependency Review`, so consider:
1. just remove `push` action like this
2. separate them for different **trigger logic** (may need check the action doc) OR just divide into 2 CI files

**Pull request merged**